### ISSUE TITLE
Is there class right now?

### ISSUE THREAD

**Issue by dhimmel** [chaptr-global/blockchainfundamentals-penn#1]
In JMHH F94?

**Comment by trang1618** [comment]
Yup. Per my [instagram story](https://photos.app.goo.gl/VrnBdynzgxp9DNV76).

**Comment by dhimmel** [comment]
CC @cyrilmichino did the schedule change?

**Comment by dhimmel** [comment]
Unfortunately, the course has been canceled, as per the following message:

> Hi class, 
> 
> Due to an unexpected administrative issue, we are forced to cancel the Penn Blockchain Course this semester. We are sorry for the inconvenience that this causes you and hope that you will continue to stay involved with the blockchain community. The Penn Blockchain Club will offer courses again next semester that we hope you are able to join. In the meantime, if you are interested in self-studying, here is a great list of resources put together by a member of the Penn Blockchain Club. 
> 
> Finally, for your inconvenience, we would love to offer you a discount to the Penn Blockchain Conference. Please apply the discount code XXXXXXXXXXX for 50% off of the student ticket prices. Visit our website for more information about the conference. 
> 
> Once again, we apologize for this inconvenience. Please reach out if you have any questions. 
> 
> Best, 
> Aaron Diamond-Reivich
